memlab.bib

```diff
@@ -11735,7 +11815,7 @@ @incollection{LyonEtal06
 	Publisher = {Edizioni Goliardiche},
 	Title = {{An ACT-R Model of Spatial Visualization Capacity}},
 	Year = {2006},
-	Bdsk-File-1 = {YnBsaXN0MDDSAQIDBFxyZWxhdGl2ZVBhdGhZYWxpYXNEYXRhXxAfMjAwNmEtbHlvbl9ndW56ZWxtYW5uX2dsdWNrLnBkZk8RAeQAAAAAAeQAAgAADW5vdXZlYXVNYWNQcm8AAAAAAAAAAAAAAAAAAMnnRuRIKwAAAxIpcB8yMDA2YS1seW9uX2d1bnplbG1hbm5fZ2x1Y2sucGRmAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADHM10zUQa4wAAAAAAAAAAAAEAAgAACSAAAAAAAAAAAAAAAAAAAAAIbWFnZWxsYW4AEAAIAADJ538kAAAAEQAIAADNRGEzAAAAAQAQAxIpcAAIxScACHtwAACYoAACAFZub3V2ZWF1TWFjUHJvOlVzZXJzOgByb2JlcnRzZWt1bGVyOgBEcm9wYm94OgBtYWdlbGxhbjoAMjAwNmEtbHlvbl9ndW56ZWxtYW5uX2dsdWNrLnBkZgAOAEAAHwAyADAAMAA2AGEALQBsAHkAbwBuAF8AZwB1AG4AegBlAGwAbQBhAG4AbgBfAGcAbAB1AGMAawAuAHAAZABmAA8AHAANAG4AbwB1AHYAZQBhAHUATQBhAGMAUAByAG8AEgBEVXNlcnMvcm9iZXJ0c2VrdWxlci9Ecm9wYm94L21hZ2VsbGFuLzIwMDZhLWx5b25fZ3VuemVsbWFubl9nbHVjay5wZGYAEwABLwAAFQACABT//wAAAAgADQAaACQARgAAAAAAAAIBAAAAAAAAAAUAAAAAAAAAAAAAAAAAAAIu}}
+	Bdsk-File-1 = {YnBsaXN0MDDUAQIDBAUGJCVYJHZlcnNpb25YJG9iamVjdHNZJGFyY2hpdmVyVCR0b3ASAAGGoKgHCBMUFRYaIVUkbnVsbNMJCgsMDxJXTlMua2V5c1pOUy5vYmplY3RzViRjbGFzc6INDoACgAOiEBGABIAFgAdccmVsYXRpdmVQYXRoWWFsaWFzRGF0YV8QHzIwMDZhLWx5b25fZ3VuemVsbWFubl9nbHVjay5wZGbSFwsYGVdOUy5kYXRhTxEB5AAAAAAB5AACAAANbm91dmVhdU1hY1BybwAAAAAAAAAAAAAAAAAAyedG5EgrAAADEilwHzIwMDZhLWx5b25fZ3VuemVsbWFubl9nbHVjay5wZGYAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAMczXTNRBrjAAAAAAAAAAAAAQACAAAJIAAAAAAAAAAAAAAAAAAAAAhtYWdlbGxhbgAQAAgAAMnnfyQAAAARAAgAAM1EYTMAAAABABADEilwAAjFJwAIe3AAAJigAAIAVm5vdXZlYXVNYWNQcm86VXNlcnM6AHJvYmVydHNla3VsZXI6AERyb3Bib3g6AG1hZ2VsbGFuOgAyMDA2YS1seW9uX2d1bnplbG1hbm5fZ2x1Y2sucGRmAA4AQAAfADIAMAAwADYAYQAtAGwAeQBvAG4AXwBnAHUAbgB6AGUAbABtAGEAbgBuAF8AZwBsAHUAYwBrAC4AcABkAGYADwAcAA0AbgBvAHUAdgBlAGEAdQBNAGEAYwBQAHIAbwASAERVc2Vycy9yb2JlcnRzZWt1bGVyL0Ryb3Bib3gvbWFnZWxsYW4vMjAwNmEtbHlvbl9ndW56ZWxtYW5uX2dsdWNrLnBkZgATAAEvAAAVAAIAFP//AACABtIbHB0eWiRjbGFzc25hbWVYJGNsYXNzZXNdTlNNdXRhYmxlRGF0YaMdHyBWTlNEYXRhWE5TT2JqZWN00hscIiNcTlNEaWN0aW9uYXJ5oiIgXxAPTlNLZXllZEFyY2hpdmVy0SYnVHJvb3SAAQAIABEAGgAjAC0AMgA3AEAARgBNAFUAYABnAGoAbABuAHEAcwB1AHcAhACOALAAtQC9AqUCpwKsArcCwALOAtIC2QLiAucC9AL3AwkDDAMRAAAAAAAAAgEAAAAAAAAAKAAAAAAAAAAAAAAAAAAAAxM=}}
```

Comment:
Should remove

Reply:
File used to create LyonEtal06. Removed.